### ISSUE TITLE
Endpoint changes for Horizon v0.17

### ISSUE THREAD

**Issue by Synesso** [Synesso/scala-stellar-sdk#57]
```
Deprecations
    • Root > protocol_version will be deprecated in v0.17.0. It is replaced by current_protocol_version and core_supported_protocol_version.
    • Ledger > transaction_count will be deprecated in v0.17.0.
    • Signer > public_key will be deprecated in v0.17.0.
```

**Comment by Synesso** [comment]
v0.17 went live and broke several function. This is now urgent. However, I'm blocked by https://stellar.stackexchange.com/questions/2238/asset-endpoint-returns-404-on-standalone-network

**Comment by Synesso** [comment]
Fixed in v0.5.2
